third-party/llvm/Makefile

```diff
@@ -36,6 +36,18 @@ LLVM_CONFIG_FILE := $(LLVM_INSTALL_DIR)/bin/llvm-config
 LLVM_CLANG_FILE := $(LLVM_INSTALL_DIR)/bin/clang
 LLVM_CLANG_CONFIG_FILE := $(LLVM_INSTALL_DIR)/configured-clang-sysroot-arguments
 
+# Sometimes one is using a custom version of GCC. In that event,
+# we'd like the built clang to use that version's std C++ headers, rather
+# than only finding the versions in /usr/include. To do that, pass
+# GCC_INSTALL_PREFIX for GCC builds.
+CHPL_LLVM_GCC_PREFIX := ""
+ifneq (, $(findstring gnu,$(CHPL_MAKE_HOST_COMPILER)))
+  CHPL_LLVM_GCC_PREFIX_DIR := $(shell ./find-gcc-prefix.sh $(CC))
+  ifneq ($(CHPL_COMM_DEBUG),)
```

Comment:
`CHPL_COMM_DEBUG` => `CHPL_LLVM_GCC_PREFIX_DIR`?

Reply:
It seems I'm on a roll here making some stupid mistakes. I'll fix, but this strategy by itself doesn't work on an XC anyway.

Reply:
Not sure I'm following, what's not working on an XC?

Reply:
/opt/gcc/6.1.0/snos/include/g++ is the path, but normal GCC installations use *c++* as that final directory. Clang doesn't know to look in *g++*.